src/test/java/com/wikia/webdriver/testcases/mercurytests/NavigationTest.java

```diff
@@ -50,8 +50,8 @@ public void validSearchPhraseTriggersSuggestionThatCanBeOpened() {
     navigation.selectSearchSuggestion(1);
   }
 
-  @Test(groups = "navigation-invalidSearchPhraseDoesntTriggerSuggestionAndSearchCanBeCancelled")
-  public void invalidSearchPhraseDoesntTriggerSuggestionAndSearchCanBeCancelled() {
+  @Test(groups = "mercury_navigation_invalidSearchPhraseDoesntTriggerSuggestionAndSearchCanBeCancelled")
+  public void mercury_navigation_invalidSearchPhraseDoesntTriggerSuggestionAndSearchCanBeCancelled() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)